packages/react-scripts/scripts/utils/createJestConfig.js

```diff
@@ -30,14 +30,14 @@ module.exports = (resolve, rootDir, isEjecting) => {
     // Then these can always stay as require.resolve()s.
     resolver: isEjecting
       ? 'jest-pnp-resolver'
-      : require.resolve('jest-pnp-resolver'),
+      : require.resolve('./pnpResolver.js'),
     setupFiles: [
       isEjecting
         ? 'react-app-polyfill/jsdom'
         : require.resolve('react-app-polyfill/jsdom'),
     ],
 
-    setupTestFrameworkScriptFile: setupTestsFile,
+    setupFilesAfterEnv: setupTestsFile ? [setupTestsFile] : [],
```

Comment:
Maybe rather than this here you modify the assignment of `setupTestsFile`  above to potentially allow multiple files to be assigned:

```js
function setupTestsFiles(rootDir) {
    const setupTestsFiles = [];
    if(fs.existsSync(paths.testsSetup)){
      const setupTestsMatches = paths.testsSetup.match(/src[/\\]setupTests\.(.+)/);
      const setupTestsFileExtension = (setupTestsMatches && setupTestsMatches[1]) || 'js';
      setupTestsFiles.push(`${rootDir}/src/setupTests.${setupTestsFileExtension}`)
    }
    return setupTestsFiles
}

...
module.exports = (resolve, rootDir, isEjecting) => {
...

setupFilesAfterEnv: setupTestsFiles(rootDir),
```

Reply:
I can totally do this, although i think that most users would use `setupFilesAfterEnv` like:

```javascript
setupFilesAfterEnv: [
  'jest-dom/extend-expect',
  'react-testing-library/cleanup-after-each'
]
```

rather than creating multiple file themselves.

IMO we should allow overriding `setupFilesAfterEnv` in `package.json` if we really want to support this.